test/modules/shadowing/issue-19167-1-f.good

```diff
@@ -1,4 +1,5 @@
-issue-19167-1-f.chpl:18: In function 'main':
-issue-19167-1-f.chpl:19: error: ambiguous call 'f()'
-issue-19167-1-f.chpl:2: note: candidates are: f()
-issue-19167-1-f.chpl:6: note:                 f()
+issue-19167-1-f.chpl:10: error: 'x' has multiple definitions in this scope
+issue-19167-1-f.chpl:11: note: it was first defined through the 'use' statement here
+issue-19167-1-f.chpl:3: note: leading to the definition here
+issue-19167-1-f.chpl:12: note: redefined through the 'use' statement here
+issue-19167-1-f.chpl:7: note: leading to the definition here
```

Comment:
It looks like there are a number of other tests in this directory with the same issue, so I'll just point that out here rather than commenting on them individually.

Reply:
The Dyno scope resolver doesn't, but the resolution pass does, printing "ambiguous call". Reading issue 19167, I think it's not good that this test deviates from the previous behavior (it was working "as [Michael] would expect" at the time of closing #19167.